src/bridge.cpp

```diff
@@ -1,24 +1,133 @@
 /** Copyright (C) 2013 Ultimaker - Released under terms of the AGPLv3 License */
 #include "bridge.h"
 
-#include "sliceDataStorage.h"
-
 namespace cura {
 
-int bridgeAngle(Polygons outline, const SliceLayer* prevLayer, Polygons& supportedRegions)
+int bridgeAngle(const Polygons& skinOutline, const SliceLayer* prevLayer, const SupportLayer* supportLayer, Polygons& supportedRegions, const double supportThreshold)
 {
-    AABB boundaryBox(outline);
+    AABB boundaryBox(skinOutline);
+
     //To detect if we have a bridge, first calculate the intersection of the current layer with the previous layer.
     // This gives us the islands that the layer rests on.
     Polygons islands;
+
+    Polygons prevLayerOutline; // we also want the complete outline of the previous layer
```

Comment:
The reason I have used thisFormat is because the original code in that function uses thatFormat for all of it's variables. If I use this_format, it will look rather out_of_place?

Reply:
We're using the boy scout rule for this: Once you change a bit of code then it should be changed. We'll not ask from pull requests to refactor our code but when adding new variables it should be correct_at_least.